index.html

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MY PAGE</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--Navigation Menu-->
+    <!-- MENU WRAPPER-->
+    <nav id="navigation">
+      <!-- THE HAMBURGER "&#9776" <-- macht die drei Striche)-->
+      <label for="hamburger">&#9776;</label>
+      <!--this is the hamburger button-->
+      <input type="checkbox" id="hamburger" />
+
+      <div id="hamburger-items">
+        <a href="#top">Home</a>
+        <a href="#Aboutme">About me</a>
+        <a href="#Career">Career</a>
+        <a href="#CV">CV</a>
+        <a href="#Languages">Skills</a>
+        <a href="#Contact">Contact</a>
+      </div>
+      <hr />
+    </nav>
+
+    <hr class="hr-top" />
+
+    <!--Intro Box & Intro Picture-->
+    <hr />
+    <section class="flex">
+      <div class="flex-intro">
+        <h1>Hi! I'm Eliane.</h1>
+
+        <h2>
+          Marketing Manager
+          <br />
```

Comment:
The br tag is actually deprecated! We can achieve the same result using just paragraph tags (because they are set to display: block; as default)